code/_onclick/hud/alert.dm

```diff
@@ -1071,3 +1097,9 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 	master_ref = null
 	owner = null
 	screen_loc = ""
+
+/// Called when someone tries to examine the alert.
+/atom/movable/screen/alert/proc/alert_examine(mob/clicker)
```

Comment:
why are we not just overriding examine and calling it above
failing that, why not make this a getter, having it both get the text and print it seems a little redundant if we do it for every override